PR Davide_Branch INTO main

DavideBrambilla98 opened the pull request:
Il risultato del branch:  https://davidebrambilla.altervista.org/

Ci sono tutte e 4 le tabelle con le relative barre di ricerca.
E' presente anche una buona base per il CSS
Manca la parte del CRUD e decidere come gestire la tabella PatologiaRicovero

DavideBrambilla98 committed:
Modifica gestione DB

DavideBrambilla98 committed:
Sistemate le tabelle

Corretti i titoli delle tabelle per poter essere usati con i vari linguaggi

DavideBrambilla98 committed:
Aggiunte le pagine per le altre tabelle

DavideBrambilla98 committed:
Modifica alla gestione del DB

Aggiunte le query per poter filtrare le altre tabelle

DavideBrambilla98 committed:
Piccole modifiche al foglio di stile e alla navigazione

DavideBrambilla98 committed:
Update style.css

DavideBrambilla98 committed:
Modificato foglio di stile e footer

DavideBrambilla98 committed:
Update style.css

DavideBrambilla98 committed:
Update DataSet.ods

Aggiunta la tabella ricoveri fatta da @Marco-Saini

Co-Authored-By: Marco-Saini <145766844+Marco-Saini@users.noreply.github.com>

DavideBrambilla98 committed:
Aggiunta connessione tra pagina html e tabella ricoveri

Marco-Saini commented:
@DavideBrambilla98 bel lavoro l'unica cosa c'é un conflitto sul file ods se non l'hai modificato(ma solo tipo ingrandito le colonne) di non includerlo nei cambiamenti oppure di risolvere il conflitto

DavideBrambilla98 committed:
Aggiunti JOIN con altre tabelle

Aggiunti i JOIN con le altre tabelle come richiesto da consegna e visualizzati nella pagina web

CONSEGNA:
se una tabella A è legata da una relazione (0:1) o (1:1) ad un’altra tabella B sarebbe il caso di mostrare insieme alla riga di A i dati caratteristici di B (oltre all’ID).

DavideBrambilla98 committed:
Aggiunti i conteggi dei ricoveri per ospedali e cittadini

Aggiunto conteggio (e mostrato nella tabella) del numero di ricoveri sia per gli ospedali che per i cittadini.

CONSEGNA:
se una tabella A è legata da una reazione (0:n) o (1:n) ad un’altra tabella B allora mostrate il numero di entità di B legate ad ogni riga di A.

DavideBrambilla98 committed:
Definizione dei metodi per impostare le chiavi e i vincoli del DB

Bisgona ancora richiamare le varie funzione per ogni tabella

DavideBrambilla98 committed:
Piccoli cambiamenti grafici

DavideBrambilla98 committed:
Sistemata query dei filtri

Corretto problema che non filtrava nel modo giusto a causa del GROUP BY

Marco-Saini commented:
Top
